addons/website_event_crm_questions/models/event_registration.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    def _get_lead_description_registration(self, prefix='', line_suffix=''):
+        """Add the questions and answers linked to the registrations into the description of the lead."""
+        reg_description = super(EventRegistration, self)._get_lead_description_registration(prefix=prefix, line_suffix=line_suffix)
+        if not self.registration_answer_ids:
+            return reg_description
+
+        answer_descriptions = []
+        for answer in self.registration_answer_ids:
+            answer_value = answer.value_answer_id.name if answer.question_type == "simple_choice" else answer.value_text_box
```

Comment:
we just have simple choice ? no multiple_choice nor matrix ? nor numeric or date fields in the event questions ?